adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -444,6 +486,42 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] {
   }
 
   /**
+   * Performs a broadcast right outer join between this RDD and data that has been broadcast.
+   *
+   * In a broadcast join, the left RDD is collected to the driver,
+   * and broadcast to all the nodes in the cluster (broadcastTree). The key equality
+   * function used for this join is the reference region overlap function. Since this
+   * is a right outer join, all values in the left RDD that do not overlap a
+   * value from the right RDD are dropped. If a value from the right RDD does
+   * not overlap any values in the left RDD, it will be paired with a `None`
+   * in the product of the join. As compared to broadcastRegionJoin, this function allows the
```

Comment:
...still getting lost by left and right here.  One of the two mentions of RDD should be replaced with broadcast data.